website/project/views/node.py

```diff
@@ -692,7 +692,7 @@ def _view_project(node, auth, primary=False):
             'date_created': iso8601format(node.date_created),
             'date_modified': iso8601format(node.logs.latest().date) if node.logs.exists() else '',
             'tags': list(node.tags.values_list('name', flat=True)),
-            'children': bool(node.nodes_active),
+            'children': bool(set(Node.objects.get_children(node)) & set(node.nodes_active)),
```

Comment:
This change appears to have broken [tests](https://travis-ci.org/CenterForOpenScience/osf.io/jobs/201399990#L1904-L1919). Previously, it was causing the [frontend](https://github.com/CenterForOpenScience/osf.io/blob/914a6e64890a7d69dbf7dbb757c85e480f9d9eea/website/templates/project/project_base.mako#L106) to [display](https://github.com/CenterForOpenScience/osf.io/blob/develop/website/static/js/pages/project-settings-page.js#L86-L87) a "`Any child components must be deleted prior to deleting this project`" warning without checking to see if those children were pointers, as  described in [the ticket](https://openscience.atlassian.net/browse/OSF-7492). Further investigation confirmed that said behavior is present on prod.